pyartcd/pyartcd/pipelines/release_from_fbc.py

```
@@ -0,0 +1,724 @@
+        self.working_dir = self.runtime.working_dir.absolute()
+        self.elliott_working_dir = self.working_dir / "elliott-working"
+        self._shipment_data_repo_dir = self.working_dir / "shipment-data-push"
+
+        # Shipment repository configuration
+        self.gitlab_url = self.runtime.config.get("gitlab_url", "https://gitlab.cee.redhat.com")
+        self.gitlab_token = None
+        self.shipment_mr_url = None
+
+        # Require shipment_path for this pipeline
+        if not shipment_path:
+            raise ValueError("--shipment-path is required for the release-from-fbc pipeline")
```

Ruff (0.14.3)

Avoid specifying long messages outside the exception class

(TRY003)